ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Resources.hs

```diff
@@ -63,25 +66,43 @@ data SharedResources m =
 -- "Test.Consensus.PeerSimulator.ScheduledChainSyncServer".
 data ChainSyncResources m =
   ChainSyncResources {
-    -- | A mailbox of node states that is updated by the scheduler in the peer's active tick,
-    -- waking up the chain sync server.
-    csrNextState :: StrictTMVar m NodeState,
-
     -- | The current known intersection with the chain of the client.
     csrCurrentIntersection :: StrictTVar m (Point TestBlock),
 
     -- | The final server passed to typed-protocols.
-    csrServer :: ChainSyncServer (Header TestBlock) (Point TestBlock) (Tip TestBlock) m ()
+    csrServer :: ChainSyncServer (Header TestBlock) (Point TestBlock) (Tip TestBlock) m (),
+
+    -- | This action blocks while this peer is inactive in the point schedule.
+    csrTickStarted :: STM m ()
```

Comment:
Although it's accurate, it's funny to say “this action blocks” when it is called `*started` :p

Reply:
yes, I am very open for better naming and/or phrasing 😅